pkg/kubelet/container.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubelet
+
+import (
+	"hash/adler32"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubelet/rocket"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/types"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+	"github.com/golang/glog"
+)
+
+const defaultEndpoint = "/home/yifan/gopher/src/github.com/coreos/rocket/bin/rkt"
+
+var rkt *rocket.RocketRuntime
+
+func init() {
+	var err error
+	rkt, err = rocket.NewRocketRuntime(defaultEndpoint)
+	if err != nil {
+		panic(err)
+	}
+}
+
+// FindContainerInPod finds the container in the pod.
```

Comment:
This style of docstring that just repeats what the method is named is really just white noise. Either beef up the description or drop it altogether.